StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/SA1609SA1610CodeFixProvider.cs

```diff
@@ -50,7 +50,8 @@ public override Task RegisterCodeFixesAsync(CodeFixContext context)
                 }
 
                 string description = DocumentationResources.SA1609SA1610CodeFix;
-                context.RegisterCodeFix(CodeAction.Create(description, cancellationToken => this.GetTransformedDocumentAsync(context.Document, diagnostic, cancellationToken)), diagnostic);
+                context.RegisterCodeFix(CodeAction.Create(description, cancellationToken => this.GetTransformedDocumentAsync(context.Document, diagnostic, cancellationToken),
+                    equivalenceKey: SA1619GenericTypeParametersMustBeDocumentedPartialClass.DiagnosticId + SA1610PropertyDocumentationMustHaveValueText.DiagnosticId), diagnostic);
```

Comment:
:question: Can we get away with just using `string.Empty` for all cases where we only register one fix for each diagnostic? The advantage of this as a convention is it will be immediately obvious looking at the call to `CodeAction.Create` whether we intend to provide only one fix or multiple potential fixes.

:bulb: If not `string.Empty`, at least simplify the pattern to just `nameof(SA1609SA1610CodeFixProvider)` as the equivalence key.